addons/stock/i18n/stock.pot

```diff
@@ -217,12 +217,7 @@ msgid "<p class=\"oe_view_nocontent_create\">\n"
 "                Click to add a serial number.\n"
 "              </p><p>\n"
 "                This is the list of all the production lots you recorded. When\n"
-"                you select a lot, you can get the \n"
-"                traceability of the products contained in lot. By default, the\n"
-"                list is filtered on the serial numbers that are available in\n"
-"                your warehouse but you can uncheck the 'Available' button to\n"
-"                get all the lots you produced, received or delivered to\n"
-"                customers.\n"
+"                you select a lot, you can get the traceability of the products contained in lot.\n"
```

Comment:
The issue is that this was removed at 3b58ee833e098b061f603fd8c31b4b42e9c0488c but it shouldn't have been merged in 8.0 as it may break existing translations. To keep it translated for the one that have not updated the views, keep both terms.